cinema/urls.py

```diff
@@ -18,6 +18,8 @@
 router.register("movie_sessions", MovieSessionViewSet)
 router.register("orders", OrderViewSet)
 
-urlpatterns = [path("", include(router.urls))]
+urlpatterns = [path("", include(router.urls)),
+               path("movies/", views.MovieViewSet.as_view({"get": "list"}), name="movie-list"),
+]
```

Comment:
The path 'movies/' on line 22 is redundant because the router already registers the 'movies' endpoint on line 17. This duplication can lead to confusion and should be removed unless there is a specific reason to have both.